lib/routers/profile/person.js

```diff
@@ -185,7 +185,8 @@ const getPil = (req, res, next) => {
   const { pil, trainingPils, pilLicenceNumber } = req.profile;
 
   if (!pilLicenceNumber) {
-    return next(new NotFoundError());
+    res.response = null;
+    return next();
```

Comment:
What's the theory behind _not_ responding with a 404 in this case?

Reply:
because this endpoint is hit on every profile, and if they don't have a pil it will throw a 404, which we do not want

Reply:
basically it's to allow the api to return nothing, when there is nothing to return

Reply:
👍